### ISSUE TITLE
Make install error

### ISSUE THREAD

**Issue by paulofrazao** [weaveworks/flux2-openshift#13]
Trying to build new Operator and with the latest version of 0.34.0, I get the following error:

make release
./release.sh
Flux version: 0.34.0
Generating the manifests using the built CLI ...
Exporting gotk-components.yaml ...
make: *** [Makefile:2: release] Error 1

I also tried version 0.33.0 with no success. I went back to version 0.32 and this worked fine.

Looks like there were some changes after version 0.32.0, that is causing this issue.

https://user-images.githubusercontent.com/45883050/190167468-7b64071c-7d91-4064-bc48-50dce4a1e7b3.mp4

**Comment by paulofrazao** [comment]
Zendesk ticket #2463

[](https://weavesupport.zendesk.com/agent/tickets/2463)

**Comment by makkes** [comment]
Taking a look.

**Comment by makkes** [comment]
@paulofrazao what is the error you're seeing?

**Comment by paulofrazao** [comment]
When executing the test.sh script, we get the error below.
![image](https://user-images.githubusercontent.com/45883050/196232228-25025482-312a-47a0-a3fa-ff518803dc3f.png)

Attached is the error log.
[746434899.zip](https://github.com/weaveworks/flux2-openshift/files/9802703/746434899.zip)

**Comment by makkes** [comment]
@paulofrazao I'm not sure if we're looking at the same code but the test.sh script [doesn't use kind but ekz](https://github.com/weaveworks/flux2-openshift/blob/9a45898cdc503eb01ff703db464213273716a522/test.sh#L4).

**Comment by makkes** [comment]
The reason the custom-8g62d pod is failing is this:

```
2022-10-11T17:07:46.734677245Z stderr F Error: open db-704484487: permission denied
2022-10-11T17:07:46.734850632Z stderr F Usage:
2022-10-11T17:07:46.734863867Z stderr F   opm registry serve [flags]
2022-10-11T17:07:46.734866321Z stderr F 
2022-10-11T17:07:46.734868205Z stderr F Flags:
2022-10-11T17:07:46.734869888Z stderr F   -d, --database string          relative path to sqlite db (default "bundles.db")
2022-10-11T17:07:46.734871631Z stderr F       --debug                    enable debug logging
2022-10-11T17:07:46.734873344Z stderr F   -h, --help                     help for serve
2022-10-11T17:07:46.734874937Z stderr F   -p, --port string              port number to serve on (default "50051")
2022-10-11T17:07:46.734876981Z stderr F       --skip-migrate             do  not attempt to migrate to the latest db revision when starting
2022-10-11T17:07:46.734878424Z stderr F   -t, --termination-log string   path to a container termination log file (default "/dev/termination-log")
2022-10-11T17:07:46.734879816Z stderr F       --timeout-seconds string   Timeout in seconds. This flag will be removed later. (default "infinite")
2022-10-11T17:07:46.734881099Z stderr F 
2022-10-11T17:07:46.734882532Z stderr F Global Flags:
2022-10-11T17:07:46.734884255Z stderr F       --skip-tls   skip TLS certificate verification for container image registries while pulling bundles or index
2022-10-11T17:07:46.734885517Z stderr F 
```
```

**Comment by paulofrazao** [comment]
> 

The script is modified to remove ekz and and add kind.

**Comment by stefanprodan** [comment]
I have no clue what ekz is and I have no bandwidth to understand and maintain this distribution. If we can't find someone with OpenShift knowledge to maintain this repo, let's accept defeat and delete Flux from OpenShift. I see no value in having a Flux version on OperatorHub which is months behind upstream and is affected by major CVEs.

**Comment by sjavage** [comment]
@paulofrazao have we taken this to any of our RH contacts and gotten their input?

**Comment by stefanprodan** [comment]
@sjavage given that this repo uses custom tools like Typescript generators and EKZ which is a project owned by @chanwit, I don’t see how RH could help fix this.

**Comment by chanwit** [comment]
EKZ is just a left over.

We have been using KIND in the test since March.

**Comment by stefanprodan** [comment]
| We have been using KIND in the test since March.

But this say otherwise: https://github.com/weaveworks/flux2-openshift/blob/main/test.sh#L4

**Comment by paulofrazao** [comment]
> 

Since taking over from Chanwit, I just modify the test.sh file before execution. I will permanently change this once this issue is resolved.

**Comment by paulofrazao** [comment]
> @paulofrazao have we taken this to any of our RH contacts and gotten their input?



> @paulofrazao have we taken this to any of our RH contacts and gotten their input?

I haven't take this up with RH as they were assisting with Marketplace setup and not OpertorHub. The process is different, but we still need this addressed first as we want both the Marketplace and OperatorHub to have the same version.

**Comment by stefanprodan** [comment]
Fixed by #16
